File: Modul-2/C/PRAK205-2310817320007-ZahraNabila.c
#include <stdio.h>
#include <math.h>

int main()
{
    int sisiA, sisiB;

    scanf("%d %d", &sisiA, &sisiB);

    int Alas        = sqrt(pow(sisiB, 2) -  (pow(sisiA, 2)));
    int Tinggi      = sisiA;
    int Keliling    = sisiA + sisiB + Alas;
    int Luas        = 0.5 * Alas * sisiA;

    printf("Alas        = %d cm \n", Alas);
    printf("Tinggi      = %d cm \n", Tinggi);
    printf("Keliling    = %d cm \n", Keliling);
    printf("Luas        = %d cm^2 \n", Luas);

    return 0;
}

File: Modul-2/Python/PRAK205-2310817320007-Zahra Nabila.py
import math
sisiA = input() . split()

if len(sisiA)== 1:
    sisiA = int(sisiA[0])
    sisiB = int(input())

elif len(sisiA)== 2:
    sisiA, sisiB = map(int, sisiA)

Alas = int(math.sqrt(sisiB**2 - sisiA**2))
Tinggi = sisiA
Keliling = sisiA + sisiB + Alas
Luas = 0.5 * Alas * sisiA

print(f"Alas          = {Alas} cm ")
print(f"Tinggi        = {Tinggi} cm ")
print(f"Keliling      = {Keliling} cm ")
print(f"Luas          = {Luas:.0f} cm^2 ")